### ISSUE TITLE
Dropdown menu needs to add support for keyboard (lp-app.js)

### ISSUE THREAD

**Issue by hyperpress** [hyperpress/logicalphase#246]
`<a id="dropdownMenuButton" class="toolbar-platform-chooser__button" aria-expanded="false"> ` doesn’t end up focusable, I think due to having neither href nor tabIndex=0, so the dropdown menu isn’t reachable by keyboard. Many thanks to "@bathos whitespace" for reporting the issue on Polymer Slack Channel.
